upgrade/upgrade-scripts/upgrade-container

```diff
@@ -395,6 +395,8 @@ function create_upgrade_container() {
 	EOF
 		die "failed to create container service override file"
 
+	fix_and_migrate_services "$CONTAINER"
```

Comment:
since this function will mask the docker service, where do we unmask that service?

Reply:
Yeah, this way docker is masked straight away when we first run the container, that said I'm not convinced that is necessary. I'll try the change you proposed and move it into execute instead.